docs: KB refresh from Zendesk tickets (Jun 1 – Jul 31, 2026) (#2995)
wandb/docs: support/models/articles/how-do-i-add-more-seats.mdx

Reviewer: I think we tend to call this your AISE, but it might be worth confirming.

The W&B team uses this information to provision the additional seats on your account.

> ## How W&B counts seats

Reviewer: I'm not sure this needs an H2. I suspect that the details on line 15 should go in the intro para.

W&B assigns seats per user at the organization level, not per team, and teams inherit them. Adding a user to a second team doesn't consume another seat.

> For how seats and access levels are assigned, see [Assign or update a user's access](/platform/hosting/iam/access-management/manage-organization#assign-or-update-a-users-access).

Reviewer: If you keep this as the last sentence in the article, this edit might make sense.
```suggestion
See [Assign or update a user's access](/platform/hosting/iam/access-management/manage-organization#assign-or-update-a-users-access) for details.
```
